### ISSUE TITLE
TypeError: Object of type DataFrame is not JSON serializable

### ISSUE THREAD

**Issue by nrasif** [mljar/plotai#6]
Hi, I got an error says "TypeError: Object of type DataFrame is not JSON serializable" when I tried an example from this GitHub

```
import pandas as pd
from plotai import PlotAI
# create some data
df = pd.DataFrame({"x":[1,2,3], "y": [4,5,6]})
# do a plot
plot = PlotAI(df)
plot.make("scatter plot")
```

![image](https://github.com/mljar/plotai/assets/64336363/e56a659c-07be-427d-87f1-b1ac1a04cf6f)
![image](https://github.com/mljar/plotai/assets/64336363/cd7f88fe-1bc3-4dcf-a535-77cee4f45be4)

Any suggestion how to fix this?

**Comment by pplonski** [comment]
Hi @nrasif,

Looks like there is some bug in the code ... I will try to fix it this week.

**Comment by pplonski** [comment]
I can reproduce the issue.

**Comment by pplonski** [comment]
There was bug in the code. I fixed it and released new version `0.0.4`, please let me know if it works for you.
